### DISCUSSION TITLE
#53 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by 0xPabloLI** [tog-ETH-er/togETHer#37] [General]
Agenda:

-Community updates
-Website update for new member
-Website update for [Shared event calendar](https://calendar.google.com/calendar/u/0?cid=N2Q2NGZlNzFiZTlmNWU3Mjg5MThmMjliYjAxZTcxMDI2OTg2ZTYxN2ZmMTNmY2IwOWUyYmMyMWJhY2YxOGNjZEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t) 
-Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：
https://meet.google.com/uin-uwpc-gta

**Comment by 0xPabloLI** [comment]
PlanckerDAO

12.3 [社交登录钱包分享](https://plancker.notion.site/16-6099e0bb72fb4640a373f9df36921189)
12.6 [Antalpha Hacker House分享：开发者如何找到以太坊需求](https://plancker.notion.site/de579cee812944b7a6a6079672e0b6bc)
12.9 Matters分享：去中心化社会的历史渊源、思考的路径
12.9 [Rebase 2022 Hackathon 分享 Account Abstraction and Implementation: EIP4337](https://www.youtube.com/watch?v=3meS2sqWoUY)
12.10 [小V共读会5：Having a safe CEX](https://plancker.notion.site/17-V-5-911093aa39ad465e9a85c44d3b29fa66)

**Comment by Stamp9** [comment]
## 原语里弄

新课题发布。
https://www.primitiveslane.org/get-involved

**Comment by bambalados** [comment]
**Crypto Tech Night**

CTN#19 - MPC 钱包与Account Abstraction 12月15号

**Comment by xrdavies** [comment]
#### Rebase

- 14期 Web3极客日报
- Who-Is-Hiring 招聘若干条
- Rebase黑客松启动，已有超过100名选手报名参赛，工作坊举办四期

**Comment by Riely007** [comment]
### 翻译小组

- 官网翻译 review 进行中

**Comment by stephanieeee-ho** [comment]
### ECN
日常翻译和七日谈输出

**Comment by Emilyxiannv** [comment]
### DAOSquare
-协助支持Rebase的黑客松
-在Twitter做了ChatGPT的分享，space链接：https://twitter.com/i/spaces/1YpJkgDvgBNJj
以及文字版分享：https://app.musex.io/article/14/25

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

8个PR #891 ～ #898

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了3场分享会

- 《Element Finance 协议分析》 by sam
  - https://www.youtube.com/watch?v=sgcCPF3dHhU

- 《讲解如何提交一个EIP》by Mike & Xavi

- 《深度分析下个万亿赛道-零知识证明+去中心化计算》by 谢大炮
